code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/components/SmallImageCard.kt

```diff
@@ -49,49 +39,48 @@ fun SmallImageCard(
     observer: AepUIEventObserver?,
 ) {
 
-    // TODO - Implement the SmallImageCard composable
-    // Here code added as placeholder for reference, actual implementation is pending
-
+    // TODO - Add id for LaunchedEffect, test if it is working
     LaunchedEffect(key1 = Unit) {
         observer?.onEvent(UIEvent.Display(ui))
     }
 
+    // TODO - Add id for DisposableEffect, test if it is working
     DisposableEffect(key1 = Unit) {
         onDispose {
             observer?.onEvent(UIEvent.Dismiss(ui))
         }
     }
 
     Card(
-        shape = RoundedCornerShape(8.dp),
         modifier = Modifier
-            .fillMaxWidth()
-            .padding(16.dp)
             .clickable {
                 observer?.onEvent(UIEvent.Interact(ui, UIAction.CLICK))
-            },
-        elevation = 4.dp
+            }
     ) {
-        Row(
-            modifier = Modifier.padding(16.dp),
-            verticalAlignment = Alignment.CenterVertically
-        ) {
+        Row {
             // TODO - Add image support
-            Spacer(modifier = Modifier.width(16.dp))
-
-            Column(
-                verticalArrangement = Arrangement.Center
-            ) {
-                ui.getTemplate().title.let {
-                    Text(
-                        text = it?.content ?: "",
-                        style = style.getTitleTextStyle(ui.getTemplate()),
-                    )
-                }
-                Text(
-                    text = ui.getTemplate().body?.content ?: "",
-                    style = MaterialTheme.typography.body1
+            Column {
```

Comment:
- How are we exposing the button, and text alignments w.r.t the row and columns?
- Will you be adding Image support in a follow up PR?

Reply:
Card, Row, Column styling will be added in the next PR